docs/anonymizer/index.md

```diff
@@ -1,22 +1,191 @@
-# Presidio Analyzer
+# Presidio Anonymizer
 
 ## Description
 
-## Table of contents
+The Presidio anonymizer is a Python based module for anonymizing detected PII text
+entities with desired values.
+
+Persidio anonymizer comes with a predefined anonymizers but can easily be extended.
 
 ## Installation
 
+=== "Using pip"
+
+    !!! note "Note"
+        Consider installing the Presidio python packages on a virtual environment like venv or conda.
+    
+    To get started with Presidio-anonymizer, run:
+    
+    ```sh
+    pip install presidio-anonymizer
+    ```
+
+=== "Using Docker"
+
+    !!! note "Note"
+        This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+    
+    ```sh
+    # Download image from Dockerhub
+    docker pull mcr.microsoft.com/presidio-anonymizer
+    
+    # Run the container with the default port
+    docker run -d -p 5001:5001 mcr.microsoft.com/presidio-anonymizer:latest
+    ```
+
+=== "From source"
+
+    First, clone the Presidio repo. [See here for instructions](../installation.md#install-from-source).
+    
+    Then, build the presidio-anonymizer container:
+    
+    ```sh
+    cd presidio-anonnymizer
+    docker build . -t presidio/presidio-anonymizer
+    ```
+
 ## Getting started
 
-- Running in Python.
-- Running using Docker.
-- Creating new anonymizers.
-- API reference.
+Presidio anonymizer comes by default with the following anonymizers:
+
+| Anonymizer type | Description | Parameters
+| --- | ---| ---|
+| replace | replaces the PII with desired value | `new_value` - replaces existing text with the given value.<br> If `new_value` is not supplied or empty, default behavior will be: <entity_type\> e.g: <PHONE_NUMBER\> |
```

Comment:
This whole part can be dropped now that we have the REST API documentation? Or should we leave it since we have the 'as-a-package' usage?

Reply:
I added it for the as-a-package usage. to be more clear.